app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_clss: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_clss
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self, distance: float, power: float, rating: float, count: int
+    ) -> None:
+        self.distance_from_city_center = distance
+        self.clean_power = power
+        self.average_rating = round(rating, 1)
```

Comment:
i am not sure it's necessary to use round() here since You do rounding in rate_service method as it should be